docs/platform/howto/byoc/enable-byoc.md

```diff
@@ -23,7 +23,7 @@ eligibility for the feature.
 
```

Comment:
Can we remove or shorten the text (17-22)? The process to enable BYOC is explained in detail a bit further down the page